src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
+import {
+  Select, SelectOption,
+} from '@momentum-ui/react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+import WebexLocalMedia from '../WebexLocalMedia/WebexLocalMedia';
+
+/**
+ *
+ * @returns {object} Returns all of available cameras
+ */
+
+const getAvailableCameras = async (setDevices) => {
+  const cameraDevices = await navigator.mediaDevices.enumerateDevices()
```

Comment:
`navigator.mediaDevices` does not seem to be defined and tests are failing:
> TypeError: Cannot read property 'enumerateDevices' of undefined
    at _callee$ (/home/circleci/webex-components/src/components/WebexVideoSettings/WebexVideoSettings.jsx:16:54)

I think you can mock it in `src/storyshots.test.js` using the technique described here: https://stackoverflow.com/a/65216531/210550

But, instead of an empty value, please return an example list of devices (not just cameras!):
```js

const availableMediaDevices = [
   ...device1...,
   ...device2...,
   ...etc...
];
...
enumerateDevices: jest.fn(() => Promise.resovle(availableMediaDevices)),
```